### PR TITLE
Implement request #316, and minor ui fixes.

### PR DESCRIPTION
**Pull request by donhatch**
- Implement request #316: viewer menu now opens/closes on handle click instead of hover, just like editor.
- Menu handle icon now changes on open/close as appropriate.  Added necessary image files.
- Default is still open, auto-closing after 3 seconds.
- Explicit open or close cancels the initial auto-close timeout.
- Since Examples doesn't close on hover-out any more, there needs to be a way to close it, so made clicking on Examples toggle it instead of just opening it.
- Fix: Clicking on Examples now cancels the initial auto-close timeout.
- Fix: The initial auto-close now closes Examples too, if open.
- Fix: Editor handle icon now changes on open/close as intended (the icon was getting set on the wrong element before)

### CODE REVIEW

**Comment by z3dev** [comment]
@donhatch Wow! Thanks for,the nice changes.

Just for reference, which browsers are you testing with?

**Comment by kaosat-dev** [comment]
Very nice work @donhatch  , much appreciated !
I'll do a full review/ local test tonight.

**Comment by donhatch** [comment]
Hi @z3dev,
I tested this using chrome 61.0.3163.100 and safari 10.0,
both on a macbook pro running macos 10.12 (sierra).

**Comment by kaosat-dev** [comment]
@donhatch I tried your PR locally, and I like it a lot !
But one thing is still weird : unless you explicitly click on the handle it still opens 'on hover'
I think it would feel more natural if after the initial closing after the timeout, it would only open/close with explicit clicks.

**Comment by donhatch** [comment]
Hi @kaosat-dev, I agree it shouldn't open on hover any more; I'm surprised if it's still doing that for you.
I don't think it's doing that for me, on the browsers I mentioned above.
And I thought I removed the lines that implement open-on-hover, in style.css, didn't I?
So it's very strange if it's still doing it for you; are you sure?

**Comment by kaosat-dev** [comment]
Hi @donhatch double checked after your message & turns out it is the Chrome's caching that was messing things up! 
Testing in Chrome, Safari and Firefox on MacOs.
Will merge asap

File: src/ui/index.js [hunk 1]
```diff
@@ -86,32 +86,42 @@ function init () {
     const versionText = `Version ${version}`
     document.getElementById('menuVersion').innerHTML = versionText
 
-    setTimeout(function () {
+    let initialMenuHidingTimeoutID = setTimeout(function () {
+      initialMenuHidingTimeoutID = null;
```
Comment (kaosat-dev) [inline src/ui/index.js]: The only issue I see before merging is the use of semicolons here and there.
Sorry but as per our CONTRIBUTING.md , we use [standardjs style](https://standardjs.com/)
(we will likely add some automated linting like in the IO package very soon)

File: src/ui/index.js [hunk 2]
```diff
@@ -86,32 +86,42 @@ function init () {
...
 
       // -- Examples
       examplesTitle.addEventListener('click', function (e) {
-        setElementHeight(examples, 'auto')
-        examples.style.display = 'inline'
-      // setElementHeight(options, 0)
-      // options.style.display = 'none'
+        if (initialMenuHidingTimeoutID !== null) {
+          clearTimeout(initialMenuHidingTimeoutID);
+          initialMenuHidingTimeoutID = null;
```
Comment (kaosat-dev) [inline src/ui/index.js]: ditto

File: src/ui/index.js [hunk 3]
```diff
@@ -125,6 +135,29 @@ function init () {
           list[i].addEventListener('click', onLoadExampleClicked)
       }
     }
+
+    let menuHandle = document.getElementById('menuHandle')
+    menuHandle.addEventListener('click', function (e) {
+      if (initialMenuHidingTimeoutID !== null) {
+        clearTimeout(initialMenuHidingTimeoutID);
```
Comment (kaosat-dev) [inline src/ui/index.js]: etc

File: src/ui/index.js (hunk 1 above)
Comment (donhatch) [reply src/ui/index.js]: fixed.

File: src/ui/index.js (hunk 2 above)
Comment (donhatch) [reply src/ui/index.js]: fixed.

File: src/ui/index.js (hunk 3 above)
Comment (donhatch) [reply src/ui/index.js]: fixed

**Comment by donhatch** [comment]
@kaosat-dev , I fixed the semicolons.
I guess my two commits should be squashed into one-- I'm not sure whether that's something that I should do somehow, or whether you will do it during the pull?

**Comment by kaosat-dev** [comment]
@donhatch thanks ! It is fine , we always use 'squash and merge' strategy, so we can ensure a single commit per PR and standardised final commit names (that get used by tooling down the line for things such as changelog generation, etc)

**Pull request merged**